File: Api/rest_assured/src/test/java/support/utils/GetProperties.java
package support.utils;

import java.io.FileInputStream;
import java.io.IOException;
import java.util.Properties;

import static support.utils.Environments.getEnvironments;

public class GetProperties {


    private static final String DATA_CONFIG = "src/test/resources/";
    private static Properties properties;

    public static String getProp(String value) {
        try {
            if (properties == null) {
                GetProperties.properties = new Properties();
                GetProperties.properties.load(new FileInputStream(DATA_CONFIG + getEnvironments() +".properties"));
            }

        } catch (IOException e) {
            e.printStackTrace();
        }

        return GetProperties.properties.getProperty(value);
    }
}

File: Api/rest_assured_kotlin/src/test/java/support/utils/GetProperties.kt
package support.utils

import support.utils.Environments.environments
import java.util.Properties
import java.io.FileInputStream
import java.io.IOException

object GetProperties {
    private const val DATA_CONFIG = "src/test/resources/"
    private var properties: Properties? = null
    @JvmStatic
    fun getProp(value: String?): String {
        try {
            if (properties == null) {
                properties = Properties()
                properties!!.load(FileInputStream(DATA_CONFIG + environments + ".properties"))
            }
        } catch (e: IOException) {
            e.printStackTrace()
        }
        return properties!!.getProperty(value)
    }
}